### PR TITLE
Route meta data made available in twig templates through the gimme object

### PR DESCRIPTION
**Pull request by djbrd-sourcefabric**
| Q             | A
| ------------- | ---
| Bug fix?      | [no]
| New feature?  | [yes]
| BC breaks?    | [no]
| Deprecations? | [no]
| Tests pass?   | [yes]
| Fixed tickets | [SWP-239]
| License       | AGPLv3

Route meta data has been made accessible in the twig templates through the gimme object

### CODE REVIEW

File: src/SWP/Bundle/ContentBundle/Loader/MetaLoader.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    public function __construct(
+        $configurationPath,
+        CacheProvider $metadataCache)
+    {
+        $this->configurationPath = $configurationPath;
+        $this->metadataCache = $metadataCache;
+    }
+
+    protected function getConfiguration()
```
Comment (takeit) [inline src/SWP/Bundle/ContentBundle/Loader/MetaLoader.php]: `{@inheritdoc}` is missing

File: src/SWP/Bundle/WebRendererBundle/Controller/ContentController.php [hunk 2]
```diff
@@ -29,6 +29,10 @@ public function renderPageAction(Request $request, $contentTemplate, $contentDoc
             $this->container->get('context')->registerMeta('article', $articleMeta);
         }
 
+        if (null != $routeMeta) {
```
Comment (takeit) [inline src/SWP/Bundle/WebRendererBundle/Controller/ContentController.php]: why not `null !== $routeMeta` ?

File: src/SWP/Bundle/ContentBundle/Loader/RouteLoader.php [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+     * @return bool
+     */
+    public function isSupported($type)
+    {
+        return self::SUPPORTED_TYPE === $type;
+    }
+
+    private function getRouteMeta($route)
+    {
+        if (!is_null($route)) {
```
Comment (takeit) [inline src/SWP/Bundle/ContentBundle/Loader/RouteLoader.php]: `null !== $route`

**Comment by ahilles107** [comment]
closed in favour #118

**Pull request closed without merging**